specutils/analysis/flux.py

```diff
@@ -97,7 +97,7 @@ def _compute_line_flux(spectrum, regions=None):
         calc_spectrum = spectrum
 
     # Average dispersion in the line region
-    avg_dx = (np.abs(np.diff(calc_spectrum.spectral_axis))).quantity
+    avg_dx = (np.abs(np.diff(calc_spectrum.spectral_axis)))
```

Comment:
Just FYI, this change is because most ufuncs applied to ``SpectralCoord`` now return ``Quantity`` except for a few explicitly specified cases that make physical sense.